src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -145,4 +145,41 @@ public function getDebug()
 
         return $this->debug;
     }
+
+    /**
+     * Returns some or all the existing flash messages. The method is variadic:
+     * if no arguments are passed, all flashes are returned; if one or more
+     * arguments are passed, only the flashes that belong to that category are
+     * returned; e.g. getFlashes('notice') or getFlashes('notice', 'error').
```

Comment:
Also we could support array of categories? i.e: `getFlashes(['notice', 'error'])`, sometimes this information is configurable.